### PR TITLE
APP-60: Fix wildcards

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

**Comment by forGGe** [comment]
Looks good to me

**Comment by forGGe** [comment]
cc: @dyosick @RostakaGmfun

File: stormdemo/source/c/build.bat [hunk 1]
```diff
@@ -30,7 +30,7 @@ set LIBS_PATH=libs
 set KAA_LIB_PATH=%LIBS_PATH%\kaa
 set KAA_C_LIB_HEADER_PATH=%KAA_LIB_PATH%\src
 set KAA_CPP_LIB_HEADER_PATH=%KAA_LIB_PATH%\kaa
-set KAA_SDK_TAR="kaa-client*.tar.gz"
+set KAA_SDK_TAR="kaa-c*.tar.gz"
```
Comment (ghost) [inline stormdemo/source/c/build.bat:33]: If I don't mistake, as for Kaa 0.8.0 C SDK doesn't support Windows out of the box.
So how do you think. maybe we can just remove this bat-script?

**Comment by RostakaGmfun** [comment]
LGTM

**Pull request merged**